src/instructlab/eval/mt_bench.py

```diff
@@ -94,27 +94,30 @@ class MTBenchEvaluator(AbstractMTBenchEvaluator):
 
     name = "mt_bench"
 
-    def gen_answers(self, server_url) -> None:
+    def gen_answers(self, server_url, api_key=None) -> None:
```

Comment:
nit: I know it hasn't been done for the other args yet, but would you mind making these api_key: str | None = None

Reply:
updated